### ISSUE TITLE
Improve contrast of colours

### ISSUE THREAD

**Issue by marien-probesys** [Probesys/bileto#97]
Contrasts should be good enough for the text thanks to https://www.radix-ui.com/colors. However, the contrasts are pretty bad to distinguish components from the background.

Reference: https://www.w3.org/TR/WCAG21/#non-text-contrast

> The visual [presentation](https://www.w3.org/TR/WCAG21/#dfn-presentation) of the following have a [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) of at least 3:1 against adjacent color(s):
>
> - User Interface Components: Visual information required to identify [user interface components](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components) and [states](https://www.w3.org/TR/WCAG21/#dfn-states), except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;
> - Graphical Objects: Parts of graphics required to understand the content, except when a particular presentation of graphics is [essential](https://www.w3.org/TR/WCAG21/#dfn-essential) to the information being conveyed.

**Comment by marien-probesys** [comment]
Related to https://github.com/Probesys/bileto/issues/562

**Comment by marien-probesys** [comment]
Recommendations:

- replace `--color-slate2: #f9f9fb` by `#f6f6f9`
- use a dark color on primary buttons, or tweak the value of `--color-teal9`
